cinema/models.py

```diff
@@ -35,12 +39,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> pathlib.Path:
+    filename = (
+        f"{slugify(instance.title)}"
+        f"-{uuid.uuid4()}" + pathlib.Path(filename).suffix
+    )
+    return pathlib.Path("upload/titles") / pathlib.Path(filename)
```

Comment:
suggest you use os.path.join to avoid errors are related to the use of different OS